README.md

```diff
@@ -10,6 +10,8 @@ OCamlFormat is probably what you are after!
 
 OCamlFormat works by parsing then outputting again the same OCaml source file in a consistent style.
 
+Read the [manual](https://v3.ocaml.org/p/ocamlformat/0.21.0/doc/index.html) to learn more about OCamlFormat!
```

Comment:
This is a better link I think:
```suggestion
Read the [documentation](https://v3.ocaml.org/p/ocamlformat) to learn more about OCamlFormat!
```
It doesn't specify the version (it redirects to the latest version) and show the important things: README, opam info, homepage